src/main/scala/spray/revolver/DebugSettings.scala

```diff
@@ -2,7 +2,7 @@ package spray.revolver
 
 case class DebugSettings(port: Int = 5005, suspend: Boolean = false) {
   def toCmdLineArg: String =
-    "-agentlib:jdwp=transport=dt_socket,server=y,suspend=%s,address=%d".format(b2str(suspend), port)
+    "-agentlib:jdwp=transport=dt_socket,server=y,suspend=%s,address=*:%d".format(b2str(suspend), port)
```

Comment:
Hmm, would this open the port on `0.0.0.0`?

If you'd do that on your host system, it'd mean the port would be open for anyone who can connect to it. I expect the debugging interface allows arbitrary code execution, so perhaps this should be an opt-in option instead?

Reply:
Good point, I've updated the code to have it as an optional parameter